src/Quarrel/App.xaml

```diff
@@ -5,74 +5,118 @@
+                    <Windows10FallCreatorsUpdate:AcrylicBrush x:Key="AcrylicSubFrameBackground"
                                   BackgroundSource="Backdrop"
                                   TintColor="#FFD0D0D0"
                                   TintOpacity="0.7"
                                   FallbackColor="#FFD0D0D0"/>
 
-                    <AcrylicBrush x:Key="AcrylicFlyoutBackground"
+                    <Windows10FallCreatorsUpdate:AcrylicBrush x:Key="AcrylicFlyoutBackground"
                                   BackgroundSource="Backdrop"
                                   TintColor="#DDFFFFFF"
                                   TintOpacity="0.8"
                                   FallbackColor="#DDFFFFFF"/>
 
-                    <AcrylicBrush x:Key="AcrylicUserBackground"
+                    <Windows10FallCreatorsUpdate:AcrylicBrush x:Key="AcrylicUserBackground"
                                   BackgroundSource="Backdrop"
                                   TintColor="#FFF2F2F3"
                                   TintOpacity="0.85"
                                   FallbackColor="#FFF2F2F3"/>
 
-                    <AcrylicBrush x:Key="AcrylicUserBackgroundDarker"
+                    <Windows10FallCreatorsUpdate:AcrylicBrush x:Key="AcrylicUserBackgroundDarker"
                                   BackgroundSource="Backdrop"
                                   TintColor="#FFE0E0E5"
                                   TintOpacity="0.3"
                                   FallbackColor="#FFE0E0E5"/>
 
-                    <AcrylicBrush x:Key="AcrylicButtonBackground"
+                    <Windows10FallCreatorsUpdate:AcrylicBrush x:Key="AcrylicButtonBackground"
                                   BackgroundSource="Backdrop"
                                   TintColor="#DDFFFFFF"
                                   TintOpacity="0.4"
                                   FallbackColor="#DDFFFFFF"/>
-                    
-                    <AcrylicBrush x:Key="SubFrameHostBackgroundBrush"
+
+                    <Windows10FallCreatorsUpdate:AcrylicBrush x:Key="SubFrameHostBackgroundBrush"
                                   BackgroundSource="Backdrop"
                                   TintColor="#A0FFFFFF"
                                   TintOpacity="0.7"
                                   FallbackColor="#A0FFFFFF"/>
 
+                    <SolidColorBrush x:Key="LightBG" Color="#f6f7f9"/>
+                    <SolidColorBrush x:Key="MidBG" Color="#e3e8ed"/>
+                    <SolidColorBrush x:Key="MidBG_hover" Color="#2e3036"/>
+                    <SolidColorBrush x:Key="MidBG_focus" Color="#303238"/>
+                    <SolidColorBrush x:Key="DarkBG" Color="#e1e3e5"/>
+                    <SolidColorBrush x:Key="LinkColor" Color="#0096cf"/>
+                    <SolidColorBrush x:Key="InvertedBG" Color="#161b2b"/>
+                </Windows10FallCreatorsUpdate:ResourceDictionary>-->
```

Comment:
Quarrel should maintain acrylic support on light mode, perhaps there should be four themes, light acrylic, dark acrylic, light and dark.

Reply:
Windows themes don't work that way.  There are only three themes, Light, Dark and HighContrast.  We need to move to WinUI to get acrylic in Mobile and restore to the previous theme scheme.